ui/src/components/Modal/__tests__/Modal.test.ts

```diff
@@ -0,0 +1,64 @@
+import { mount } from "@vue/test-utils";
+import Modal from "../Modal.vue";
+import { CreateElement, VNode } from "vue";
+
+describe("Modal", () => {
+  // Both Vue.config.async and mountOptions.sync are not working?
+  // Vue.config.async = false;
+  // expect(Vue.config.async).toBe(false);
+
+  const title = "Modal Title";
+  const wrapper = mount(Modal, {
+    // does not work. infact it is default value
+    // https://github.com/vuejs/vue-test-utils/pull/1062
+    sync: true,
```

Comment:
This should work for `prop.sync` (https://github.com/vuejs/vue-test-utils/pull/1062) but no luck

Reply:
> The modal works now in docs.
> But the test-utils doesn't seems working `prop.sync`, even after I set the `sync: true` during mounting. In this case, I am still updating the `active` prop manually.
> 
> @ChristianKienle Should this PR be kept as multiple commits since there is a merge from other branch? Or perhaps combine as single commit.

I think you can squash the commits and turn multiple commits into a single one. You have the option to select that option by clicking on the arrow of the merge button.